src/index.html

```diff
@@ -1,12 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.1.1/css/all.css" />
+    <link rel="icon" type="image/x-icon"
+        href="https://www.pngkey.com/detail/u2r5i1r5q8q8q8i1_favicon-clipart-cafe-restaurant-food/">
+    <title>MealsDB</title>
 </head>
+
 <body>
-    <h1>hello world!</h1>
+    <header>
+        <nav>
+            <img src="" alt="logo">
```

Comment:
I suggest we should add a logo to make the `UI` more attractive and user friendly👏